api/src/chat/dialogs/myanmar/disaster-assessment/review/reviewAffectedPeopleDialog.ts

```diff
@@ -26,55 +24,60 @@ export const reviewAffectedPeopleDialog: RcdaChatDialog = rcdaChatDialog(
         references: () => [askAffectedPeople.askAffectedPeopleDialog]
     });
 
-function createAffectedPeopleReviewCard(session: RcdaTypedSession):Message {
+const editAction = "edit";
```

Comment:
Let's ensure that the declarations are always at the top after imports. Easier to find :)